web/cobrands/fixmystreet/assets.js

```diff
@@ -416,28 +393,62 @@ fixmystreet.add_assets = function(options) {
                 }
                 zoom_to_assets(asset_layer);
             } else {
-                asset_layer.setVisibility(false);
+                if (!asset_layer.fixmystreet.always_visible) {
```

Comment:
Should you wrap the whole `on()` change function in this check - presumably you don't want any of it to run when there's a category change?